src/lib/microsoft-partnercenter.ts

```diff
@@ -0,0 +1,154 @@
+import { AxiosInstance } from 'axios'
```

Comment:
Out of curiosity, why are some of these `import type` and some not? All could be just `import { ..everything } from './types`.